src/components/FAB.js

```diff
@@ -64,15 +66,17 @@ class FAB extends PureComponent {
         });
 
         return (
-            <AnimatedPressable
-                onPress={this.props.onPress}
-                style={[
-                    styles.floatingActionButton,
-                    {transform: [{rotate}], backgroundColor},
-                ]}
-            >
-                <AnimatedIcon src={Plus} fill={fill} />
-            </AnimatedPressable>
+            <KeyboardAvoidingView behavior={Platform.OS === 'ios' ? 'position' : 'height'}>
```

Comment:
Just a heads up this is not implemented correctly. Can you create a follow up PR to address this? @aliabbasmalik8 

https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions